src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformAppCompatTextView.java

```diff
@@ -0,0 +1,61 @@
+package com.microsoft.maui;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.AppCompatTextView;
+
+public class PlatformAppCompatTextView extends AppCompatTextView {
+    public PlatformAppCompatTextView(@NonNull Context context) {
+        super(context);
+    }
+
+    private boolean isFormatted;
+
+    /**
+     * Sets isFormatted=false and calls super.setText(). This is required as setText() is final.
+     * @param text
+     */
+    public final void setPlainText(CharSequence text) {
+        isFormatted = false;
+        super.setText(text);
+    }
+
+    /**
+     * Implemented so C# will generate a PlainText property
+     * @return
+     */
+    public final CharSequence getPlainText() {
+        return super.getText();
+    }
+
+    /**
+     * Sets isFormatted=true, used for HTML text content
+     * @param text
+     * @param type
+     */
+    @Override
+    public void setText(CharSequence text, BufferType type) {
+        isFormatted = true;
```

Comment:
This line isn't right as `setPlainText()` above actually calls into here. Looking into it...